### PR TITLE
Portofolio

### PR DESCRIPTION
**Pull request by hassanosamaa**

### CODE REVIEW

File: src/template/partials/header.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            <div class="container">
+                
+                <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon">
+                        <img src="assets/images/bar.svg" alt="bar">
+                    </span>
+                </button>
+
+                <h1>
+                <a class="navbar-brand" href="#header-sec">
```
Comment (mukhtar-ali) [inline src/template/partials/header.html]: The logo anchor should redirect to homepage

File: src/template/partials/header.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                    <img class="d-none d-md-block" src="assets/images/LOGO.png" alt="logo">
+                    <img class="d-md-none" src="assets/images/LOGO-sm.png" alt="logo">
```
Comment (mukhtar-ali) [inline src/template/partials/header.html]: Why did you duplicate the logo, you should control the size with width or height and I prefer the height

File: src/template/partials/header.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+                </a>
+                </h1>
+
+                <form class="d-flex  d-lg-none" role="search">
+                    <button type="submit">Hire Me</button>
+                </form>
```
Comment (mukhtar-ali) [inline src/template/partials/header.html]: What is this?

File: src/pages/index.html [hunk 4]
```diff
@@ -1,6 +1,330 @@
 {% extends "layout.html" %}
```
Comment (mukhtar-ali) [inline src/pages/index.html]: Missing page title

File: src/pages/index.html [hunk 5]
```diff
@@ -1,6 +1,330 @@
...
+                    <div class="info text-center text-lg-start">
+                       <h4>Hi I am</h4> 
+                       <h3>Mahmood Fazile</h3>
+                       <h2>UI/UX designer</h2>
+                       <div class="social justify-content-center justify-content-lg-start">
+                            <div>
+                                <a href="#">    
+                                  <img src="assets/images/social1.svg" alt="insta">
+                                </a>
+                            </div>
```
Comment (mukhtar-ali) [inline src/pages/index.html]: You should style the anchor instead of the div now the image only is clickable

File: src/pages/index.html [hunk 6]
```diff
@@ -1,6 +1,330 @@
...
+                            <div>
+                                <a href="#">    
+                                    <img src="assets/images/social4.svg" alt="be">
+                                  </a>
+                            </div>
+                           
+                       </div>
+                       <div class="btn justify-content-center justify-content-lg-start">
+                            <button class="btn1">Hire Me</button>
+                            <button class="btn2">Downlead CV </button>
```
Comment (mukhtar-ali) [inline src/pages/index.html]: ??

File: src/pages/index.html [hunk 7]
```diff
@@ -1,6 +1,330 @@
...
+{% import 'component/cardportofolio.html' as card3 %}
+
 {% block content %}
-<h1>master</h1>
+
+
+
+<section id="header-sec" class="header">
+    <div class="mini-header">
+
```
Comment (mukhtar-ali) [inline src/pages/index.html:14]: Background color is black #121212

File: src/pages/index.html [hunk 8]
```diff
@@ -1,6 +1,330 @@
...
+                           
+                       </div>
+                       <div class="btn justify-content-center justify-content-lg-start">
+                            <button class="btn1">Hire Me</button>
+                            <button class="btn2">Downlead CV </button>
+                       </div>
+                       
+                       <div class="experiences text-start">
+                            <div class="exp-1">
+                                <h4><span>5+</span>Experiences</h4>
```
Comment (mukhtar-ali) [inline src/pages/index.html]: This is not a title it should be a <p> or <span>

File: src/pages/index.html [hunk 9]
```diff
@@ -1,6 +1,330 @@
...
+    
+                       <img class="shape d-lg-none" src="assets/images/shap-sm.svg" alt="">
+                       <img class="shape d-none d-lg-block " src="assets/images/shap.svg" alt="">
+                    </div>
+                </div>
+    
+                <div class="col-lg-6 offset-lg-1 ">
+                    <div class="person">
+                        <img class="person w-100 d-lg-none" src="assets/images/person-sm.png" alt="person">
+                        <img class="person w-100 d-none d-lg-block" src="assets/images/person.png" alt="person">
```
Comment (mukhtar-ali) [inline src/pages/index.html]: Why the duplication?

File: src/pages/index.html [hunk 10]
```diff
@@ -1,6 +1,330 @@
...
+                        <h2>
+                            Services
+                        </h2>
+                    </div>
+                </div>
+                <div class="col-12">
+                    <div class="text-center">
+                        <p class="des-services">Lorem ipsum dolor sit amet consectetur. Imperdiet convallis blandit felis ligula aliquam</p>
+                    </div>
+                </div>
```
Comment (mukhtar-ali) [inline src/pages/index.html]: ??

File: src/pages/index.html [hunk 11]
```diff
@@ -1,6 +1,330 @@
...
+                            About Me
+                        </h2>
+                    </div>
+                </div>
+                <div class="col-12">
+                    <div>
+                        <p>
+                            User Interface and User Experience and Also video editing 
+                        </p>
+                    </div>
```
Comment (mukhtar-ali) [inline src/pages/index.html]: ???

File: src/pages/index.html [hunk 12]
```diff
@@ -1,6 +1,330 @@
...
+                                </p>
+                                <button>
+                                    <span>
+                                        <img src="assets/images/Download.svg" alt="download">
+                                    </span>
+                                    Download CV
+                                </button>
```
Comment (mukhtar-ali) [inline src/pages/index.html]: ??

File: src/template/component/cardportofolio.html [hunk 13]
```diff
@@ -0,0 +1,17 @@
+{% macro card (imgPath,title,cate) %}
+
+<div class="col-10 col-md-6 col-lg-4">
+    <div class="card">
+        <a href="#">
+            <div class="img">
+                <img src="{{imgPath}}" alt="">
+            </div>
+            <div class="card-bottom d-flex justify-content-between ">
+                <h3>{{title}}</h3>
```
Comment (mukhtar-ali) [inline src/template/component/cardportofolio.html:10]: This should b anchor

File: src/template/component/cardportofolio.html [hunk 14]
```diff
@@ -0,0 +1,17 @@
+{% macro card (imgPath,title,cate) %}
+
+<div class="col-10 col-md-6 col-lg-4">
+    <div class="card">
+        <a href="#">
+            <div class="img">
+                <img src="{{imgPath}}" alt="">
+            </div>
```
Comment (mukhtar-ali) [inline src/template/component/cardportofolio.html]: This should be anchor

File: src/template/component/cardportofolio.html [hunk 15]
```diff
@@ -0,0 +1,17 @@
+{% macro card (imgPath,title,cate) %}
+
+<div class="col-10 col-md-6 col-lg-4">
+    <div class="card">
+        <a href="#">
```
Comment (mukhtar-ali) [inline src/template/component/cardportofolio.html]: Remove this

File: src/pages/index.html [hunk 16]
```diff
@@ -1,6 +1,330 @@
...
+                                <div>
+                                    <input class="form-control"   id="emailInput" type="text" placeholder="Email">
+                                    <select id="Service" class="select-selected">
+                                        <option value="volvo" selected>Service of Interest</option>
+                                        <option value="saab">Saab</option>
+                                        <option value="vw">VW</option>
+                                        <option value="audi" >Audi</option>
+                                      </select>
+                                    <textarea  id="detailsInput" type="text" placeholder="Project Details..."></textarea>
+                                    <button>Send</button>
```
Comment (mukhtar-ali) [inline src/pages/index.html]: ????

File: src/pages/index.html [hunk 17]
```diff
@@ -1,6 +1,330 @@
...
+                            </div>
+                            <div class="col-11 col-lg-5 text-center  text-lg-end">
+                                <div>
+                                    <input class="form-control"   id="emailInput" type="text" placeholder="Email">
+                                    <select id="Service" class="select-selected">
+                                        <option value="volvo" selected>Service of Interest</option>
+                                        <option value="saab">Saab</option>
+                                        <option value="vw">VW</option>
+                                        <option value="audi" >Audi</option>
+                                      </select>
```
Comment (mukhtar-ali) [inline src/pages/index.html]: Not like the design

File: src/pages/index.html [hunk 18]
```diff
@@ -1,6 +1,330 @@
...
+                                            {{card3.card(item.imgPath,item.title,item.cate)}}
+                                    {% endfor %}
+                                </div>
+                            </div>
+                            <div class="row row-non-slider justify-content-center ">
+                                
+                                {% for item in dataEn.portofolio %}
+                                        {{card3.card(item.imgPath,item.title,item.cate)}}
+                                {% endfor %}
+                            </div>
```
Comment (mukhtar-ali) [inline src/pages/index.html:202]: You shouldn't duplicate the code.

**Review by mukhtar-ali — CHANGES_REQUESTED** [review]

File: src/template/partials/header.html (hunk 1 above)
Comment (hassanosamaa) [reply src/template/partials/header.html]: done

File: src/template/partials/header.html (hunk 2 above)
Comment (hassanosamaa) [reply src/template/partials/header.html]: done

File: src/template/partials/header.html (hunk 3 above)
Comment (hassanosamaa) [reply src/template/partials/header.html]: done

File: src/pages/index.html (hunk 4 above)
Comment (hassanosamaa) [reply src/pages/index.html]: done

File: src/pages/index.html (hunk 5 above)
Comment (hassanosamaa) [reply src/pages/index.html]: done

File: src/template/component/cardportofolio.html (hunk 14 above)
Comment (hassanosamaa) [reply src/template/component/cardportofolio.html]: done

File: src/template/component/cardportofolio.html (hunk 15 above)
Comment (hassanosamaa) [reply src/template/component/cardportofolio.html]: done

File: src/pages/index.html (hunk 16 above)
Comment (hassanosamaa) [reply src/pages/index.html]: done

File: src/pages/index.html (hunk 17 above)
Comment (hassanosamaa) [reply src/pages/index.html]: done

File: src/pages/index.html (hunk 6 above)
Comment (hassanosamaa) [reply src/pages/index.html]: done

File: src/pages/index.html (hunk 7 above)
Comment (hassanosamaa) [reply src/pages/index.html:14]: done

File: src/template/component/cardportofolio.html (hunk 13 above)
Comment (hassanosamaa) [reply src/template/component/cardportofolio.html:10]: done

File: src/pages/index.html (hunk 9 above)
Comment (hassanosamaa) [reply src/pages/index.html]: done

File: src/pages/index.html (hunk 11 above)
Comment (hassanosamaa) [reply src/pages/index.html]: done

File: src/pages/index.html (hunk 12 above)
Comment (hassanosamaa) [reply src/pages/index.html]: done

File: src/pages/index.html (hunk 10 above)
Comment (hassanosamaa) [reply src/pages/index.html]: done

File: src/pages/index.html (hunk 8 above)
Comment (hassanosamaa) [reply src/pages/index.html]: done

File: src/template/partials/header.html (hunk 1 above)
Comment (Ahmed-Hassan-96) [reply src/template/partials/header.html]: #home-page !!!!!!

File: src/pages/index.html (hunk 4 above)
Comment (Ahmed-Hassan-96) [reply src/pages/index.html]: What about Website Name

File: src/pages/index.html (hunk 5 above)
Comment (Ahmed-Hassan-96) [reply src/pages/index.html]: Why you manage z-index to social anchors and buttons?

File: src/pages/index.html (hunk 6 above)
Comment (Ahmed-Hassan-96) [reply src/pages/index.html]: anchor not button

File: src/pages/index.html (hunk 8 above)
Comment (Ahmed-Hassan-96) [reply src/pages/index.html]: smae issue like social icons  anchor inside div

File: src/pages/index.html (hunk 10 above)
Comment (Ahmed-Hassan-96) [reply src/pages/index.html]: Not done

File: src/pages/index.html (hunk 11 above)
Comment (Ahmed-Hassan-96) [reply src/pages/index.html]: Not done

File: src/pages/index.html (hunk 12 above)
Comment (Ahmed-Hassan-96) [reply src/pages/index.html]: text must be inside span

File: src/pages/index.html (hunk 5 above)
Comment (hassanosamaa) [reply src/pages/index.html]: Because when I placed the svg, it covered me

File: src/pages/index.html (hunk 4 above)
Comment (hassanosamaa) [reply src/pages/index.html]: Changed name

File: src/pages/index.html (hunk 6 above)
Comment (hassanosamaa) [reply src/pages/index.html]: done

File: src/pages/index.html (hunk 10 above)
Comment (hassanosamaa) [reply src/pages/index.html]: i delete col-12 Repeated

File: src/pages/index.html (hunk 11 above)
Comment (hassanosamaa) [reply src/pages/index.html]: i delete col-12 Repeated

**Pull request closed without merging**